### DISCUSSION TITLE
Example of post-deploy hook script

### DISCUSSION THREAD

**Discussion by tyctor** [inspircd/inspircd#2085] [Support]
Hi

in changelog of 3.17.0 release is 
```
Added an example post-deploy hook script for use with CertBot, Dehydrated, or other ACME SSL (TLS) renewal tools.
```

can you please point me where to find that hook/script?

thanks and i am looking forward to your answer

**Comment by genius3000** [comment]
[deploy-ssl.sh](https://github.com/inspircd/inspircd/blob/insp3/make%2Ftemplate%2Fdeploy-ssl.sh)  

That's the _template_ version of the script, path variables are set correctly upon install. It should be installed in your InspIRCd _run_ directory. Depending on how you installed InspIRCd that could be a system folder like `/etc/inspircd` or `/var/run/inspircd`, and so forth. Of course you can always use the template and modify the path variables appropriately.
